tools/webpack/packages.js

```diff
@@ -82,15 +88,47 @@ const bundledPackagesPhpConfig = [
 	},
 } ) );
 
-const gutenbergPackages = Object.keys( dependencies )
-	.filter(
-		( packageName ) =>
-			! BUNDLED_PACKAGES.includes( packageName ) &&
-			packageName.startsWith( WORDPRESS_NAMESPACE ) &&
-			! packageName.startsWith( WORDPRESS_NAMESPACE + 'react-native' ) &&
-			! packageName.startsWith( WORDPRESS_NAMESPACE + 'interactivity' )
-	)
-	.map( ( packageName ) => packageName.replace( WORDPRESS_NAMESPACE, '' ) );
+const NOT_WP_SCRIPTS = [
```

Comment:
Similar to script modules, maybe a field in a package's package.json can indicate it should be included.

Probably not just this list (copied from the devDependencies on trunk)